### ISSUE TITLE
The plugin doesn't work on CraftBukkit

### ISSUE THREAD

**Issue by DoNotSpamPls** [DoNotSpamPls/SkinsRestorerX#20]
```
[09:26:46 ERROR]: Error occurred while enabling SkinsRestorer v13.2-SNAPSHOT (Is                                                                                                                                                                                                it up to date?)
java.lang.NullPointerException
        at skinsrestorer.bukkit.SkinsRestorer.log(SkinsRestorer.java:49) ~[?:?]
        at skinsrestorer.bukkit.SkinsRestorer.onEnable(SkinsRestorer.java:106) ~                                                                                                                                                                                               [?:?]
        at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:321) ~[c                                                                                                                                                                                               raftbukkit-1.8.8.jar:git-Bukkit-e1ebe52]
        at org.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader                                                                                                                                                                                               .java:332) [craftbukkit-1.8.8.jar:git-Bukkit-e1ebe52]
        at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManage                                                                                                                                                                                               r.java:404) [craftbukkit-1.8.8.jar:git-Bukkit-e1ebe52]
        at org.bukkit.craftbukkit.v1_8_R3.CraftServer.loadPlugin(CraftServer.jav                                                                                                                                                                                               a:342) [craftbukkit-1.8.8.jar:git-Bukkit-e1ebe52]
        at org.bukkit.craftbukkit.v1_8_R3.CraftServer.enablePlugins(CraftServer.                                                                                                                                                                                               java:314) [craftbukkit-1.8.8.jar:git-Bukkit-e1ebe52]
        at net.minecraft.server.v1_8_R3.MinecraftServer.s(MinecraftServer.java:4                                                                                                                                                                                               06) [craftbukkit-1.8.8.jar:git-Bukkit-e1ebe52]
        at net.minecraft.server.v1_8_R3.MinecraftServer.k(MinecraftServer.java:3                                                                                                                                                                                               70) [craftbukkit-1.8.8.jar:git-Bukkit-e1ebe52]
        at net.minecraft.server.v1_8_R3.MinecraftServer.a(MinecraftServer.java:3                                                                                                                                                                                               25) [craftbukkit-1.8.8.jar:git-Bukkit-e1ebe52]
        at net.minecraft.server.v1_8_R3.DedicatedServer.init(DedicatedServer.jav                                                                                                                                                                                               a:235) [craftbukkit-1.8.8.jar:git-Bukkit-e1ebe52]
        at net.minecraft.server.v1_8_R3.MinecraftServer.run(MinecraftServer.java                                                                                                                                                                                               :504) [craftbukkit-1.8.8.jar:git-Bukkit-e1ebe52]
        at java.lang.Thread.run(Unknown Source) [?:1.8.0_151]
```

**Comment by DoNotSpamPls** [comment]
The issue has been found to be because of the color messages. Because of this, I have decided to remove colors from the console messages, since they tend to cause compatibility issues like this (also they don't really work on many server hosts such as Multicraft and MCMA)

**Comment by DoNotSpamPls** [comment]
Pushed out a temporary fix in commit 5fa5d3b.
In the future I'll try to fix it more permanently.

**Comment by DoNotSpamPls** [comment]
Currently working to fix it permanently!

**Comment by DoNotSpamPls** [comment]
Permanently fixed in commit 0681fca
